test/t8_test_cmesh_readmshfile.c

```diff
@@ -0,0 +1,164 @@
+#include <t8_eclass.h>
```

Comment:
After the licence text, please add a brief description of the purpose of this test.